Source/JavaScriptCore/runtime/JSModuleRecord.cpp

```diff
@@ -126,15 +126,15 @@ void JSModuleRecord::instantiateDeclarations(JSGlobalObject* globalObject, Modul
             RETURN_IF_EXCEPTION(scope, void());
             switch (resolution.type) {
             case Resolution::Type::NotFound:
-                throwSyntaxError(globalObject, scope, makeString("Indirectly exported binding name '", String(exportEntry.exportName.impl()), "' is not found."));
+                throwSyntaxError(globalObject, scope, makeString("Indirectly exported binding name '"_s, String(exportEntry.exportName.impl()), "' is not found."_s));
```

Comment:
Should be able to pass `exportEntry.exportName` without `String` and avoid the reference count churn.